// The Firmament project
// Copyright (c) 2011-2012 Malte Schwarzkopf <malte.schwarzkopf@cl.cam.ac.uk>
//
// RegistrationMessage informs about the existence of a resource.

syntax = "proto3";

package firmament;

import "base/resource_desc.proto";
import "base/resource_topology_node_desc.proto";

message RegistrationMessage {
  string uuid = 1;
  string location = 2;
  ResourceDescriptor res_desc = 3;
  ResourceTopologyNodeDescriptor rtn_desc = 4;
}
